tests/test_notanorm.py

```diff
@@ -1210,6 +968,23 @@ def test_where_or(db):
     assert len(db.select("foo")) == 3
 
 
+def test_where_complex(db):
```

Comment:
Should add tests for:

1. `Where([{"bar": db.subq(...)}])`
2. `Where([{"bar": [1, 2, 3]}])`

Reply:
ok